### PR TITLE
Add deposit and withdraw on warp account

### PR DESCRIPTION
**Pull request by dev8723**
logic mostly copied from warp web app

### CODE REVIEW

File: src/job.ts [hunk 1]
```diff
@@ -36,14 +37,6 @@ export class JobSequenceMsgBuilder {
   }
 }
 
-export const base64encode = (input: object): string => {
```
Comment (dev8723) [inline src/job.ts:39]: move to `utils/contract.ts` for now, we put all helpers that want to be exported there for now

File: src/sdk.ts [hunk 2]
```diff
@@ -256,6 +256,72 @@ export class WarpSdk {
 
     return this.wallet.tx(txPayload);
   }
+
+  public async depositAccount(sender: string, account: string, token: Token, amount: string): Promise<TxInfo> {
```
Comment (dev8723) [inline src/sdk.ts]: i'm using `string` as type of `amount` here where web app use `u<Big>`. 
this is the only difference between it and logic in web app

Comment (dev8723) [reply src/sdk.ts]: maybe i should use `Uint128` here?  cause `reward` type is `Uint128`

File: src/utils/token.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { BigSource } from 'big.js';
```
Comment (dev8723) [inline src/utils/token.ts:1]: copied from web app, i believe we want to replace all the duplicate code in web app and use sdk there instead, right?

File: src/sdk.ts (hunk 2 above)
Comment (simke9445) [reply src/sdk.ts]: it's fine, Uint128 = string

File: src/utils/token.ts (hunk 3 above)
Comment (simke9445) [reply src/utils/token.ts:1]: eventually yes, rn not so urgent

File: src/sdk.ts [hunk 4]
```diff
@@ -256,6 +256,72 @@ export class WarpSdk {
...
+    } else {
+      txPayload = TxBuilder.new()
+        .send(sender, account, { [token.denom]: amount })
+        .build();
+    }
+
+    return this.wallet.tx(txPayload);
+  }
+
+  public async withdrawAccount(owner: string, receiver: string, token: Token, amount: string): Promise<TxInfo> {
```
Comment (simke9445) [inline src/sdk.ts]: all sdk tx functions should have first param set as `sender: string` which is the address of the tx signer - in this case owner == signer

File: src/sdk.ts (hunk 2 above)
Comment (simke9445) [inline src/sdk.ts]: lets rename:
- depositAccount -> depositToAccount
- withdrawAccount -> withdrawFromAccount

**Pull request merged**